packages/editor/src/bindings/pattern-overrides.js

```diff
@@ -9,18 +9,18 @@ const CONTENT = 'content';
 export default {
 	name: 'core/pattern-overrides',
 	label: _x( 'Pattern Overrides', 'block bindings source' ),
-	getValue( { registry, clientId, attributeName } ) {
-		const { getBlockAttributes, getBlockParentsByBlockName } =
-			registry.select( blockEditorStore );
-		const currentBlockAttributes = getBlockAttributes( clientId );
+	getValue( { select, clientId, attributeName } ) {
+		const { getBlock, getBlockParentsByBlockName } =
+			select( blockEditorStore );
+		const { attributes: currentBlockAttributes } = getBlock( clientId );
```

Comment:
Why use `getBlock`? That does not contain the attributes right? We'll somehow have to make sure it's not recursive, or pass the attributes to this function.

Reply:
> Why use getBlock? That does not contain the attributes right?

From my tests, it seems it actually includes the attributes. I decided to use that because calling `getBlockAttributes` causes an infinite loop. Maybe it is better to pass the attributes as an argument or prevent the loop, as you suggest. Any preference?

Reply:
I think somehow we'll have to make `getBlockAttributes` work, someone might decide to use it and not know it's not allowed. Also, we might have to filter `getBlock` as well at some point?

Reply:
Yes, I think you're right. The simplest solution that come to my mind is having an external `isProcessingBindings` variable which by default is `false`, it is set to `true` when bindings processing starts, and it is set to `false` when bindings processing finishes. Could something like that be enough? I quickly tested it and it seems to work but I don't know if there are any implications.

> we might have to filter getBlock as well at some point?

Mmm, I guess so.

Reply:
I've tried to make it work with `getBlockAttributes` with [this logic](https://github.com/WordPress/gutenberg/pull/61817/commits/ed11e1ad90c10a063a5facf8bea4b3f50caae79c). While it works, I am not sure if it is the best approach. Could something like that work or should I look for a better solution?